app/src/main/java/com/example/trempel/network/CategoryDomainModel.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.trempel.network
+
+import com.example.trempel.R
+
+enum class CategoryDomainModel(val title: String, val image: Int) {
```

Comment:
I wonder if it can be marked as `internal`

Reply:
done